tests/moduleTests/vfShow.ts

```diff
@@ -72,6 +72,43 @@ export default function tests() {
         assert.deepEqual(renderer.vfTies[0].last_note, n2.activeVexflowNote);
     });
 
+    test('music21.vfShow.Renderer prepareTies in voices across barline', assert => {
+        const m1_sop_voice = new music21.stream.Voice();
+        const n_m1_v1 = new music21.note.Note('C5');
+        m1_sop_voice.repeatAppend(n_m1_v1, 2);
+        const m1_alto_voice = new music21.stream.Voice();
+        const n_m1_v2 = new music21.note.Note('E4');
+        m1_alto_voice.repeatAppend(n_m1_v2, 2);
+        const m1 = new music21.stream.Measure();
+        m1.append(m1_sop_voice);
+        m1.append(m1_alto_voice);
+
+        const m2_sop_voice = new music21.stream.Voice();
+        const n_m2_v1 = new music21.note.Note('C5');
+        m2_sop_voice.repeatAppend(n_m2_v1, 2);
+        const m2_alto_voice = new music21.stream.Voice();
+        const n_m2_v2 = new music21.note.Note('E4');
+        m2_alto_voice.repeatAppend(n_m2_v2, 2);
+        const m2 = new music21.stream.Measure();
+        m2.append(m1_sop_voice);
+        m2.append(m1_alto_voice);
+
+        const p = new music21.stream.Part();
+        p.append(m1);
+        p.append(m2);
+        const s = new music21.stream.Score();
+        s.append(p);
+
+        // create tie in alto
+        n_m1_v2.tie = new music21.tie.Tie('start');
```

Comment:
does repeat append create clones of notes?  I thought that it did -- or it should.  If not, we have the same note twice in the same stream.  If it does then n_m1_v2 should not exist in the stream.

Reply:
It's all good--

https://github.com/cuthbertLab/music21j/blob/0d7aa31c48f7f697c18e6c3ee7d169d56150ff9b/src/music21/base.ts#L559-L569